Makefile

```diff
@@ -117,7 +117,7 @@ run-ci:
 		-e MJKEY \
 		--memory 7500m \
```

Comment:
Do we still need to pass these here if we're passing them by via RUN_ARGS?

Reply:
`make test` passes them via RUN_ARGS because it depends on `run-headless`. This imposes the limit for `run-ci` which is a different target.

Reply:
Ah, I see.